routes/translation.py

```
@@ -0,0 +1,120 @@
+
+logger = get_logger()
+router = APIRouter(tags=["Translation"])
+
+translation_queue = []
+translation_in_progress = False
+
+@router.post("/add_to_queue/")
+async def add_to_queue(
+    target_lang: str = Form("hi-IN"),
+    output_dir: str = Form("output/"),
+    source_file: Optional[UploadFile] = File(None),
```

Ruff (0.15.15)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)